src/ray/gcs/store_client/test/store_client_test_base.h

```diff
@@ -59,32 +60,25 @@ class StoreClientTestBase : public ::testing::Test {
 
  protected:
   void Put() {
-    auto put_calllback = [this](const Status &status) {
-      RAY_CHECK_OK(status);
-      --pending_count_;
-    };
+    auto put_calllback = [this](auto) { --pending_count_; };
```

Comment:
Any reason we change binary to hex here?

Reply:
huh some strange behavior with Keys.

Basically Keys doesn't work if your prefix contains '*'/'?' those wildcard letters.

Reply:
I should probably keep the binary behavior!

Reply:
I think binary should be faster here, although not sure about the regression.